backend/src/common/services/open-ai/open-ai.service.ts

```diff
@@ -0,0 +1,25 @@
+import OpenAI from 'openai';
+import {
+    type ChatCompletion,
+    type ChatCompletionCreateParams,
+    type ChatCompletionMessageParam,
+} from 'openai/resources/index.js';
+
+class OpenAIService {
+    private openai: OpenAI;
+    public constructor(apiKey: string) {
+        this.openai = new OpenAI({ apiKey });
+    }
+
+    public async sendRequest(
+        messages: ChatCompletionMessageParam[],
+    ): Promise<ChatCompletion> {
+        const parameters: ChatCompletionCreateParams = {
+            messages,
+            model: 'gpt-3.5-turbo-0125',
```

Comment:
Maybe it would be better to move this to .env if one day we decide to use another model on prod for example